scripts/count_rosdistro_packages.py

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import print_function
```

Comment:
We don't need to keep python2 compatibility for this. I was testing it with both. Removed.